docs/sphinx/source/conf.py

```diff
@@ -324,3 +325,17 @@ def setup(app):
 # suppress "WARNING: Footnote [1] is not referenced." messages
 # https://github.com/pvlib/pvlib-python/issues/837
 suppress_warnings = ['ref.footnote']
+
+# settings for sphinx-gallery
+sphinx_gallery_conf = {
+    'examples_dirs': ['../../examples'],  # location of gallery scripts
+    'gallery_dirs': ['auto_examples'],  # location of generated output
+    # sphinx-gallery only shows plots from plot_*.py files by default:
+    # 'filename_pattern': '*.py',
+}
+# supress warnings in gallery output
+# https://sphinx-gallery.github.io/stable/configuration.html
+import warnings
```

Comment:
Not a big deal, but I'd put this import up at the top of the file since you're already making another change.